modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java

```
@@ -112,6 +113,11 @@ protected void processStatement(Statement stmnt, MessageContext msgCtx) {
                     rs.close();
                 } catch (SQLException e) {}
             }
+            if (ps != null) {
+                try {
+                    ps.close();
+                } catch (SQLException ignore) {}
```

shall we print a warning message, without ignoring the exception